doctr/models/artefacts/barcode.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import cv2
+import numpy as np
+from typing import List, Tuple
+
+__all__ = ['BarCodeDetector']
+
+
+class BarCodeDetector:
```

Comment:
For now, either we:
- convert this to a function
- add constants in the constructor (min size of barcode, canny edge args, etc)